Python/ceval.c

```diff
@@ -408,6 +415,16 @@ Py_MakePendingCalls(void)
     if (busy)
         return 0;
     busy = 1;
```

Comment:
I don't understand well the purpose of busy... is it a flag to catch reentrant call? If no, maybe it should use volalite? If yes, please add a comment on the variable declaration to document this flag.